Modules/_cursesmodule.c

```diff
@@ -5219,6 +5219,10 @@ cursesmodule_exec(PyObject *module)
     }
     SetDictInt("KEY_MIN", KEY_MIN);
     SetDictInt("KEY_MAX", KEY_MAX);
+
+#ifdef NCURSES_EXT_FUNCS
+    SetDictInt("NCURSES_EXT_FUNCS", NCURSES_EXT_FUNCS);
```

Comment:
Since it's a conditional constant, we should probably say that it's conditional in the docs, shouldn't we?